pyccl/halos/profiles.py

```diff
@@ -300,6 +300,9 @@ def convergence(self, cosmo, r, M, a_lens, a_source, mass_def=None):
                 :math:`\\kappa`
         """
         Sigma = self.projected(cosmo, r, M, a_lens, mass_def) / a_lens**2
+        if np.iterable(a_source):
```

Comment:
Small technicality, but could we replace `np.iterable` with a builtin (such as `hasattr(a_source), "__iter__")`)? The latter is twice as fast, and numpy has started to deprecate old helper functions that can easily be replaced by builtins.

Reply:
Fixed. Thanks for the explanation.

Reply:
Thanks!